tests/gpio-mock.sh

```diff
@@ -14,13 +14,11 @@ sudo apt-get update
 sudo apt-get install -y linux-headers-$(uname -r)
 sudo apt-get install -y build-essential flex bison make
 
-kernel_major_minor=$(uname -r | cut -d'.' -f1-2)
-
-echo "Kernel major minor version: $kernel_major_minor"
+echo "Kernel version: $(uname -r)"
 
 # Get GPIO Mockup driver
-wget https://raw.githubusercontent.com/torvalds/linux/v$kernel_major_minor/drivers/gpio/gpio-mockup.c
-wget https://raw.githubusercontent.com/torvalds/linux/v$kernel_major_minor/drivers/gpio/gpiolib.h
+wget https://git.launchpad.net/~canonical-kernel/ubuntu/+source/linux-azure/+git/jammy/plain/drivers/gpio/gpio-mockup.c
+wget https://git.launchpad.net/~canonical-kernel/ubuntu/+source/linux-azure/+git/jammy/plain/drivers/gpio/gpiolib.h
```

Comment:
Move `jammy` to a variable. If possible, get the value programmatically.

Reply:
Addressed on c6e5b451713e52335e2557cd99417620cbff0299, thanks for the suggestion!